### ISSUE TITLE
Log format config ignored

### ISSUE THREAD

**Issue by sronsiek** [containous/traefik#4869]
Ver: 1.7.11 - traefik_linux-arm

```
################################################################
# Global configuration
################################################################
                  
# Enable debug mode      
#
# Optional                                                      
# Default: false            
#                                                               
debug = true
              
# Log level                           
#
# Optional
# Default: "ERROR"
#           
# logLevel = "DEBUG"
logLevel = "INFO"                                                              
                                                    
# Entrypoints to be used by frontends that do not specify any entrypoint.
# Each frontend can specify its own entrypoints.
#                   
# Optional
# Default: ["http"]           
#                       
defaultEntryPoints = ["http", "https"]
[traefikLog]
                    
# Sets the filepath for the traefik log. If not specified, stdout will be used.
# Intermediate directories are created if necessary.
#                                                                        
# Optional                                      
# Default: os.Stdout
#         
# filePath = "log/traefik.log"
filePath = "traefik.log"
                                      
# Format is either "json" or "common".
#                                                               
# Optional                 
# Default: "common"                                             
#
# Is this broken (1.7.11) ?
format = "common"
```

With the above cfg, content of traefik.log is in json format. It should be in 'common' format.
I'm sure this was working correctly in 1.7.4 - just upgraded.

**Comment by traefiker** [comment]
Hi! I'm Træfiker :robot: the bot in charge of communication regulation.

Thanks for your interest in Traefik!

Issue templates help us help you by providing all necessary information.

Please edit your issue and use the available templates:

- [Feature request](https://raw.githubusercontent.com/containous/traefik/master/.github/ISSUE_TEMPLATE/Feature_request.md)
- [Bug report](https://raw.githubusercontent.com/containous/traefik/master/.github/ISSUE_TEMPLATE/Bug_report.md)

And remember: each time someone ignores the template, a cute little bunny dies.
<details>
<summary>#SaveTheCuteBunny :heart: :rabbit: :heart:</summary>

![southpark - estrella kill rabbit](https://user-images.githubusercontent.com/5674651/27143914-3cea7896-5130-11e7-81f8-02d8ac2da7a7.gif)

![southpark - estrella kill rabbit2](https://user-images.githubusercontent.com/5674651/27143924-49104c72-5130-11e7-82ae-ccedbd5dfa1c.gif)

</details>

**Comment by polarathene** [comment]
Have you tried without the `format = "common"` line? The docs only mention `json` as a valid value to provide, and that by default `common`/CLF is used. So although the intention is to explicitly set the CLF log format, it might not handle that value... but switching to json as a result would be a bug.

**Comment by dduportal** [comment]
Hi @polarathene , the PR #4953 should improve the documentation to state the followings:

- `common` is the default value, which set the log to the "CLF" (Common Log Format).
- Customizing the log format requires the user to switch to the `json` format with the current stable version of Traefik

**Comment by sronsiek** [comment]
Ok - I'm seeing the following:

```
[traefikLog]
format = "json"
```
results in the following (as expected):

```
{"level":"info","msg":"Using TOML configuration file /root/traefik/traefik_https.toml","time":"2019-06-21T18:34:42Z"}
{"level":"info","msg":"Traefik version v1.7.11 built on 2019-04-26_08:42:33AM","time":"2019-06-21T18:34:42Z"}
```
Then, **both** of the following:
```
[traefikLog]
format = "common"
```
```
[traefikLog]
# format = "common"
```
result in the same format:
```
time="2019-06-21T18:38:16Z" level=info msg="Using TOML configuration file /root/traefik/traefik_https.toml"
time="2019-06-21T18:38:16Z" level=info msg="Traefik version v1.7.11 built on 2019-04-26_08:42:33AM"
```

This is different to the json output, but I don't think this is CLF, which I believe should not have a <fieldname>="<value>" format, but look something like the following. Or is this as expected??
 ```
2019-06-21T18:38:16Z info Using TOML configuration file /root/traefik/traefik_https.toml
2019-06-21T18:38:16Z info Traefik version v1.7.11 built on 2019-04-26_08:42:33AM
```

**Comment by ldez** [comment]
`commons` is CLF only for access logs.

The `traefikLog` don't use CLF.

**Comment by dduportal** [comment]
Closed by #4953 :)
